llvm/lib/Target/AIE/AIE2Schedule.td

```diff
@@ -600,11 +600,11 @@ InstrItinData<II_LT, [InstrStage<1,  [R_WX_PORT]>], [1,1,1,1]>,
 InstrItinData<II_LTU, [InstrStage<1,  [R_WX_PORT]>], [1,1,1,1]>,
 InstrItinData<II_MOV, [PrefixCycle<P_WM_PORT>, PrefixCycle<M_WM_PORT>, PrefixCycle<DJ_WM_PORT>,
                        PrefixCycle<DN_WM_PORT>, PrefixCycle<DC_WM_PORT>, SimpleCycle<RS_WM_PORT>], [1,1]>,
-InstrItinData<II_MOV_DC, [SimpleCycle<DC_WM_PORT>], [1,1]>,
-InstrItinData<II_MOV_DJ, [SimpleCycle<DJ_WM_PORT>], [1,1]>,
-InstrItinData<II_MOV_DN, [SimpleCycle<DN_WM_PORT>], [1,1]>,
-InstrItinData<II_MOV_M, [SimpleCycle<M_WM_PORT>], [1,1]>,
-InstrItinData<II_MOV_P, [SimpleCycle<P_WM_PORT>], [1,1]>,
+InstrItinData<II_MOV_DC, [PrefixCycle<RS_WM_PORT>, SimpleCycle<DC_WM_PORT>], [1,1]>,
+InstrItinData<II_MOV_DJ, [PrefixCycle<RS_WM_PORT>, SimpleCycle<DJ_WM_PORT>], [1,1]>,
+InstrItinData<II_MOV_DN, [PrefixCycle<RS_WM_PORT>, SimpleCycle<DN_WM_PORT>], [1,1]>,
+InstrItinData<II_MOV_M, [PrefixCycle<RS_WM_PORT>, SimpleCycle<M_WM_PORT>], [1,1]>,
+InstrItinData<II_MOV_P, [PrefixCycle<RS_WM_PORT>, SimpleCycle<P_WM_PORT>], [1,1]>,
```

Comment:
Is there any use of the {DC,DJ,DN,M,P}_WM_PORT that don't also use the RS_WM_PORT?

Reply:
Also what I'm wondering, do we need all those itineraries then?

Reply:
Yes we have use of {DC,DJ,DN,M,P}_WM_PORT from few instructions from slot A like LDA*, MOVA that does not use RS_WM_PORT

Reply:
Right, so it is okay to have
```
LDA p0
NOP
NOP
NOP
NOP
NOP
MOV m0
```

but not
```
VEXTRACT p0
MOV m0
```
because they both need to book `RS_WM_PORT` at the same time.